splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java

```diff
@@ -712,6 +715,24 @@ private boolean needToUpgrade(Splice_DD_Version catalogVersion){
         return false;
     }
 
+
+    private void updateSysTableStatsView1(TransactionController tc) throws StandardException{
```

Comment:
I'm going to introduce the change we discussed earlier, adding the 'sprint number' to the version as part of the upgrade feature. Do you want to use that mechanism instead of comparing the view definition?